dev.Dockerfile

```diff
@@ -6,7 +6,11 @@ WORKDIR /code
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 RUN apt-get update \
-    && apt-get install -y --no-install-recommends curl=7.64.0-4+deb10u2 git=1:2.20.1-2+deb10u3 build-essential=12.6 \
+    && apt-get install -y --no-install-recommends \
+        curl=7.64.0-4+deb10u2 \
+        git=1:2.20.1-2+deb10u3 \
+        build-essential=12.6 \
+        postgresql-client=11+200+deb10u4 \
```

Comment:
Hm, this client is for Postgres 11 while `docker-compose.dev.yml` was switched to 12, right? Where is this client used?

Reply:
> Where is this client used?

https://github.com/PostHog/posthog/blob/8e908e6fbbe1a299efa6ecf64322388724159381/bin/tests#L12

> Hm, this client is for Postgres 11 while docker-compose.dev.yml was switched to 12, right?

This is the only available version I could find/pin with `python:3.8-slim`:
```
docker run -it --rm python:3.8-slim bash -c 'apt-get update -qq && apt-cache madison postgresql-client'
postgresql-client | 11+200+deb10u4 | http://deb.debian.org/debian buster/main amd64 Packages
postgresql-client | 11+200+deb10u3 | http://security.debian.org/debian-security buster/updates/main amd64 Packages
```

Should https://www.postgresql.org/download/linux/debian/ be used to install `psql` v12?

Reply:
I used the Postgres apt repository, but do we actually need this in the image? As in, do you run `psql` inside Docker?

Reply:
One of the benefits I see having `psql` (among other useful tools for dev, like maybe `nodemon` which is required by `bin/tests` ?) within the "dev docker image", is that it is one less thing to install/configure with the right version/host/port/etc.

```
$ docker-compose up -d
...

$ docker-compose exec web psql
psql (12.6 (Debian 12.6-1.pgdg100+1))
Type "help" for help.

posthog=# \d
                          List of relations
 Schema |                 Name                  |   Type   |  Owner
--------+---------------------------------------+----------+---------
 public | auth_group                            | table    | posthog
 public | auth_group_id_seq                     | sequence | posthog
 public | auth_group_permissions                | table    | posthog
 public | auth_group_permissions_id_seq         | sequence | posthog
```

I'm used to leveraging docker-compose to "spin up full dev environments", so I would use `psql` within the docker image.

Is there maybe a reason to not have it within that docker image?

Reply:
I could also remove the changes related to `psql` from this pull request, and open a new one dedicated to "local dev with docker-compose" ?

Reply:
Nope, thanks, this is actually convenient :D Personally I mostly use TablePlus for local dev, but this is really handy for manual queries